samples/client/src/main/java/global/covesa/sdk/client/push/ActionEvent.kt

```diff
@@ -0,0 +1,41 @@
+package global.covesa.sdk.client.push
+
+import android.app.Activity
+import global.covesa.sdk.api.client.push.PushManager
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+class ActionEvent(private val type: Type) {
+    enum class Type {
+        RegisterPush,
+        UnregisterPush,
+        SendNotification,
+    }
+
+    fun handleAction(activity: Activity) {
```

Comment:
This method should only be visible for `Activity`, so maybe move it there? Also the business logic probably belongs to `Activity`, rather than to some kind of a message queue.